src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -18,6 +26,21 @@
 (def ^:private opacity-from-default 0.5)
 (def ^:private signal-anim-duration 3900)
 (def ^:private signal-anim-duration-2 1950)
+(def ^:private min-audio-duration-ms 500)
+(def ^:private max-audio-duration-ms 120000)
+(def ^:private metering-interval 100)
+(def ^:private base-filename "am.")
+(def ^:private default-format "aac")
+
+(def ^:private record-audio-worklets (js/require "../src/js/record_audio_worklets.js"))
+(def ^:private thumb-light (js/require "../resources/images/icons2/12x12/thumb-light.png"))
+(def ^:private thumb-dark (js/require "../resources/images/icons2/12x12/thumb-dark.png"))
+
```

Comment:
also do we need to add these images into the xcode plist file as external resources?

Reply:
AFAIK is not needed, those images are bundled and served as any other image in the project